musl doesn't support _SC_LONG_BIT, just use the value from limits.h

--- src/libstatgrab/os_info.c	2015-07-28 23:27:05.860119689 -0300
+++ src/libstatgrab/os_info.c	2015-07-28 23:27:22.931906262 -0300
@@ -472,7 +472,7 @@
 		host_info_buf->bitwidth = 64;
 	}
 	else {
-		host_info_buf->bitwidth = sysconf(_SC_LONG_BIT); // well, maybe 64-bit disabled 128-bit system o.O
+		host_info_buf->bitwidth = LONG_BIT;
 	}
 	host_info_buf->host_state = sg_unknown_configuration;
 # endif
